README.md

```diff
@@ -1,14 +1,15 @@
 # Service Fabric
 
-Service Fabric is a distributed systems platform for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
+[Service Fabric is a distributed systems platform](https://github.com/GitTorre/service-fabric/tree/master/docs/architecture/README.md) for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
 
 ## Getting started with Service Fabric
 To get started with building applications for Service Fabric:
 
  - Set up your development environment for [Windows](https://docs.microsoft.com/azure/service-fabric/service-fabric-get-started), [Linux](https://docs.microsoft.com/azure/service-fabric/service-fabric-get-started-Linux), or [Mac](https://docs.microsoft.com/azure/service-fabric/service-fabric-get-started-mac).
  - [See our documentation](https://docs.microsoft.com/azure/service-fabric/).
  - [Run some sample projects](https://azure.microsoft.com/resources/samples/?sort=0&service=service-fabric).
-
+ - [Watch this.](https://www.youtube.com/watch?v=v9WHgjSqwWg) 
```

Comment:
I don't think we should add a 20 minute video about developing products on Service Fabric in the document talking about getting started with developing Service Fabric the product. I think this is a distractor for people who are here to contribute, which is what the focus of this readme should be.

Overall I think this whole section should be moved or removed.

Reply:
I'm fine with that. This codebase is gigantic, so we should try and make it a bit easier for people to understand what the code does (so, how the system works and how to use it), but I understand that point you're making. Will decrease distractions from reading, building, and  contributing code! Thanks.